### PR TITLE
Feature/atr 168 master release notes 1 3

### PR DESCRIPTION
**Pull request by k-s-popova**
Please review the release notes for 1.3.

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,22 +1,75 @@
...
+
+## Acuminator 1.3
+Acuminator 1.3 includes the diagnostics and code fixes, enhancements, and bug fixes described in this section, as well as the features that have been implemented in the previous versions.
+### New Diagnostics and Code Fixes
+In this version, diagnostics and code fixes for the following issues have been added.
+
+| Code   | Issue Description                                               | Type    | Diagnostics | Code Fix  | 
+| ------ | --------------------------------------------------------------- | ------- | ----------- | --------- | 
+| PX1012 | `PXAction` is declared on a non-primary view.                     | Warning | Available   | Available |
+| PX1015 | For a BQL statement that contains only `Required` parameters, the number of arguments of the `Select` method must be equal to the number of `Required` parameters. | Warning | Available | Unavailable |
```
Comment (SENya1990) [inline docs/ReleaseNotes.md]: @k-s-popova, the PX1015 message is not exactly valid. It seems that it was taken from the JIRA item which has outdated message. The Acuminator's wiki page is more precise. Currently the BQL statement can contain all types of parameters - Current,  Required and Optional. The supported methods are not only "Select" but also other similar methods.

Comment (k-s-popova) [reply docs/ReleaseNotes.md]: @SENya1990 , the wiki page has nothing about Current and Optional. Please update.

Comment (SENya1990) [reply docs/ReleaseNotes.md]: @k-s-popova, yes it was outdated too. Updated the wiki.

**Comment by k-s-popova** [comment]
@dropsonic, do you have anything else that is planned to be included?

**Comment by k-s-popova** [comment]
@dropsonic , 1029, 1031, 1032 added

**Pull request merged**